modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -328,7 +328,7 @@ export class Tileset3D {
 
   get queryParams(): string {
     const search = new URLSearchParams(this._queryParams).toString();
-    return search ? `?${search}` : '';
+    return search || '';
```

Comment:
I believe that search is always a string.

Reply:
Will fix after landing.